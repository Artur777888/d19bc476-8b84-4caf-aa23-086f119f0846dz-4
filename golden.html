<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
</head>
<body>
    <section>
      <img src="golden/Logo.jpg" alt="">
      <br>
     <nav>
      <ul>
        <li><a href="index.html">webstudio</a></li>
        <li><a href="mogo.html">mogo</a></li>
        <li><a href="home">home</a></li>
        <li><a href="Service">Service</a></li>
        <li><a href="Portfolio">Portfolio</a></li>
        <li><a href="About">About</a></li>
        <li><a href="Contact">Contact</a></li>
       
      </ul>
     </nav>
    </section>
   
    
    <section>
      <img src="golden/Title-Services.jpg" alt="">
      <br>


      <img src="golden/Combinedshape134.jpg" alt="">
      <h2>  E-Commerce</h2>
      <p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
      <br>

      <img src="golden/Combinedshape132.jpg" alt="">
      <h2> Responsive Web</h2>
      <p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
      <br>

      <img src="golden/Combinedshape130.jpg" alt="">
      <h2>Web Security</h2>
      <p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
      <br>
    </section>
   


    <footer>
       <h1>Title-Contact Us</h1>
        <br>

        <input type="text">your name
        <br>
        <input type="text">Your E-mail *
        <br>
        <input type="text">Subject
        <br>
        <input type="text">Your Message
        <br>

        <button>Send Message</button>




    </footer>


</body>
</html>