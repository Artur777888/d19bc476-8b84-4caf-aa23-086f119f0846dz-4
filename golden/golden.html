<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
</head>
<body>
    <section>
      <img src="Logo.jpg" alt="">
      <br>
      <button>Home</button>
      <button>Services</button>
      <button>Portfolio</button>
      <button>About</button>
      <button>Contact</button>
    </section>
   
    
    <section>
      <img src="Title-Services.jpg" alt="">
      <br>


      <img src="Combined shape 134.jpg" alt="">
      <h2>  E-Commerce</h2>
      <p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
      <br>

      <img src="Combined shape 132.jpg" alt="">
      <h2> Responsive Web</h2>
      <p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
      <br>

      <img src="Combined shape 130.jpg" alt="">
      <h2>Web Security</h2>
      <p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
      <br>
    </section>
   


    <footer>
       <h1>Title-Contact Us</h1>
        <br>

        <input type="text">your name
        <br>
        <input type="text">Your E-mail *
        <br>
        <input type="text">Subject
        <br>
        <input type="text">Your Message
        <br>

        <button>Send Message</button>




    </footer>


</body>
</html>